Week1/homework/app.js

```diff
@@ -1,11 +1,153 @@
+    author: 'Alexandre Dumas',
+  },
+  crime_and_punishment: {
+    title: 'Crime and Punishment',
+    language: 'English',
+    author: 'Fyodor Dostoyevsky',
+  },
+  the_picture_of_dorian_gray: {
+    title: 'The Picture of Dorian Gray',
+    language: 'English',
+    author: 'Oscar Wilde',
+  },
+  the_metamorphosis: {
+    title: 'The Metamorphosis',
+    language: 'English',
+    author: 'Franz Kafka',
+  },
+  the_little_prince: {
+    title: 'The Little Prince',
+    language: 'English',
+    author: 'Antoine de Saint-Exupéry',
+  },
+  lolita: {
+    title: 'Lolita',
+    language: 'English',
+    author: 'Vladimir Nabokov',
+  },
+  one_hundred_years_of_solitude: {
+    title: 'One Hundred Years of Solitude',
+    language: 'English',
+    author: 'Gabriel García Márquez',
+  },
+};
+
+// creates an object containing book covers.
+const bookCoverList = {
+  war_and_peace: './img/war-and-peace.jpg',
+  anna_karenina: './img/anna-karenina.jpg',
+  the_brothers_karamazov: './img/the-brothers-karamazov.jpg',
+  the_count_of_monte_cristo: './img/the-count-of-monte-cristo.jpg',
+  crime_and_punishment: './img/crime-and-punishment.jpg',
+  the_picture_of_dorian_gray: './img/the-picture-of-dorian-gray.jpg',
+  the_metamorphosis: './img/the-metamorphosis.jpg',
+  the_little_prince: './img/the-little-prince.jpg',
+  lolita: './img/lolita.jpg',
+  one_hundred_years_of_solitude: './img/one-hundred-years-of-solitude.jpg',
+};
+
+// creates an unordered list and adds to body.
+const listElement = document.createElement('ul');
+document.body.appendChild(listElement);
+
+// creates a function which shows book names.
+// eslint-disable-next-line no-unused-vars
+function oldBookInfo() {
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listItemOld = document.createElement('li');
+    listElement.appendChild(listItemOld);
+    listItemOld.id = 'list';
+    listItemOld.innerHTML = bookTitles[i];
+  }
 }
+
+// oldBookInfo();
+
+// creates a function which shows book details.
+function newBookInfo() {
+  // creates elements
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listItem = document.createElement('li');
+    listElement.appendChild(listItem);
+    const divElement = document.createElement('div');
+    listItem.appendChild(divElement);
+    divElement.className = 'mainDiv';
+    const bookCover = document.createElement('img');
+    divElement.appendChild(bookCover);
+    const divElementForInfo = document.createElement('div');
+    divElement.appendChild(divElementForInfo);
+    divElementForInfo.className = 'secondDiv';
+    const title = document.createElement('h2');
+    divElementForInfo.appendChild(title);
+    const writer = document.createElement('h3');
+    divElementForInfo.appendChild(writer);
+    const lang = document.createElement('p');
+    divElementForInfo.appendChild(lang);
+  }
+  // creates a method which takes information from bookDetails object.
+  const objectValue = Object.values(bookDetails);
```

Comment:
Why are you using `Object.values()`?
Because of this you are using your object as an array. Other than that you are expecting the order of your object to be the same as the order of your array, this order is not guaranteed, so you can not count on it. 
Have a look at the message in Slack from Wilgert (https://hackyourfuture.slack.com/archives/CL795TNFQ/p1566576723000500) you should use the items in your array as keys to look for in your objects. Try to change your code to this, and if something is unclear, ask a question in Slack.